src/Core/Utils/Color.hpp

```diff
@@ -37,6 +44,28 @@ class ColorBase : public Eigen::Matrix<_Scalar, 4, 1> {
     /// cast operator, mandatory to use Vector arithmetic
     operator VectorType() { return *this; }
 
+    /// convert the color expressed in sRGB color space to linear RGB
+    inline void toLinRGB( _Scalar gamma = SRGB_GAMMA) {
```

Comment:
As the method starts by `to`, I would note make it in place, and return a new instance.
Then it sounds more readable when using it:
```c++
auto sColor =  m_viewer->getBackgroundColor().tosRGB();
```
Of course you can also add `toLinRGBInPlace`, but the least we can do is to offer the aforementioned version.